### PR TITLE
Update readme for #139

### PR DESCRIPTION
**Pull request by ChrisMcKee**
add consule_encrypt_enable and _consul_raw_key to doc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -452,6 +452,16 @@ ports can be done using the `consul_ports_*` variables.
...
 
+## `consul_encrypt_enable`
+
+- Enable Gossip Encryption
+- Default value: `true`
+
+## `consul_raw_key`
+
+- Set the encryption key; should be the same across a cluster
+- Default value: ``
```
Comment (groggemans) [inline README.md]: We might want to add a note that it will be auto generated or auto retrieved from existing servers if enabled but not provided.

Comment (ChrisMcKee) [reply README.md]: @groggemans Added based on the info in encrypt_gossip.yml; didn't realise it was doing that as my vagrant clusters encrypt-keys were blank ala `"encrypt": "",` which is how I ended up digging further down.

File: README.md [hunk 2]
```diff
@@ -452,6 +452,16 @@ ports can be done using the `consul_ports_*` variables.
...
 - Default value: `server.key`
 
+## `consul_encrypt_enable`
+
+- Enable Gossip Encryption
+- Default value: `true`
+
+## `consul_raw_key`
+
+- Set the encryption key; should be the same across a cluster. If not present the key will be will be generated & retrieved from the bootstrapped server.
```
Comment (groggemans) [inline README.md]: It seems there's one `will be` to much in there.

**Comment by groggemans** [comment]
Should be good to go once the typo is fixed!

**Comment by ChrisMcKee** [comment]
Good spot

**Review by groggemans — APPROVED** [review]
LGTM

**Pull request merged**